app/src/main/java/com/cornellappdev/android/eatery/data/CafeteriaDbHelper.java

```diff
@@ -8,96 +8,100 @@
+        }
+        cursor.close();
+        return dbString;
+    }
 
-  public String getLastRow() {
-    String dbString = "";
-    SQLiteDatabase db = getWritableDatabase();
-    String query =
-        "SELECT * FROM "
-            + CafeteriaContract.CafeteriaEntry.TABLE_NAME
-            + " ORDER BY "
-            + CafeteriaContract.CafeteriaEntry._ID
-            + " DESC LIMIT 1";
-    Cursor cursor = db.rawQuery(query, null);
-    if (cursor.moveToFirst()) {
-      dbString =
-          cursor.getString(cursor.getColumnIndex(CafeteriaContract.CafeteriaEntry.COLUMN_DATA));
+    public void removeData() {
+        SQLiteDatabase db = getWritableDatabase();
+        db.execSQL(
+                "DELETE FROM "
+                        + CafeteriaContract.CafeteriaEntry.TABLE_NAME
+                        + " WHERE "
+                        + CafeteriaContract.CafeteriaEntry._ID
+                        + " =1 ");
     }
-    cursor.close();
-    return dbString;
-  }
 
-  public void removeData() {
-    SQLiteDatabase db = getWritableDatabase();
-    db.execSQL(
-        "DELETE FROM "
-            + CafeteriaContract.CafeteriaEntry.TABLE_NAME
-            + " WHERE "
-            + CafeteriaContract.CafeteriaEntry._ID
-            + " =1 ");
-  }
+    public long getProfilesCount() {
+        SQLiteDatabase db = this.getReadableDatabase();
+        long count = DatabaseUtils.queryNumEntries(db, CafeteriaContract.CafeteriaEntry.TABLE_NAME);
+        db.close();
+        return count;
+    }
 
-  public long getProfilesCount() {
-    SQLiteDatabase db = this.getReadableDatabase();
-    long count = DatabaseUtils.queryNumEntries(db, CafeteriaContract.CafeteriaEntry.TABLE_NAME);
-    db.close();
-    return count;
-  }
+    public String databaseToString() {
+        String dbString = "";
+        SQLiteDatabase db = getWritableDatabase();
+        String query = "SELECT * FROM " + CafeteriaContract.CafeteriaEntry.TABLE_NAME + " WHERE 1";
 
-  public String databaseToString() {
-    String dbString = "";
-    SQLiteDatabase db = getWritableDatabase();
-    String query = "SELECT * FROM " + CafeteriaContract.CafeteriaEntry.TABLE_NAME + " WHERE 1";
+        // Cursor point to a location in your results
+        // rawQuery can do SELECT etc
+        // CURSOR points to first in the queried result
+        Cursor c = db.rawQuery(query, null);
+        c.moveToFirst();
 
-    // Cursor point to a location in your results
-    // rawQuery can do SELECT etc
-    // CURSOR points to first in the queried result
-    Cursor c = db.rawQuery(query, null);
-    c.moveToFirst();
+        while (c.moveToNext()) {
+            if (c.getString(c.getColumnIndex(CafeteriaContract.CafeteriaEntry.COLUMN_DATA))
+                    != null) {
+                dbString +=
+                        (c.getString(
+                                c.getColumnIndex(CafeteriaContract.CafeteriaEntry.COLUMN_DATA)))
```

Comment:
you call `c.getString(c.getColumnIndex(CafeteriaContract.CafeteriaEntry.COLUMN_DATA))`, just write to a variable